src/interfaces/IWrappedMToken.sol

```diff
@@ -205,11 +220,11 @@ interface IWrappedMToken is IMigratable, IERC20Extended {
     function balanceWithYieldOf(address account) external view returns (uint256 balance);
```

Comment:
`balanceWithUnclaimedYieldOf` or even delete this function

Reply:
I will handle fucntion renames later on as:
- it is outside the scope of this PR
- they would be applicable and useful even if this pr is not merged
- the rebase(s) would potentially be too tedious at this point